### PR TITLE
Add support for positive and negative signs on numbers

### LINKED ISSUE #10: Support positive and negative signs on numbers
`+1` is a valid json5 literal, and `-1.3` as well.

### PR DESCRIPTION
**Pull request by Agent-Hellboy**
Resolves #10

### CODE REVIEW

**Comment by tusharsadhwani** [comment]
Can you format the code with black?

**Comment by tusharsadhwani** [comment]
Also add a test for a negative number.

File: src/json5kit/__init__.py [hunk 1]
```diff
@@ -279,7 +279,10 @@ def parse_number(self) -> tuple[str, float]:
         # TODO: exponent syntax support
         # TODO: Hexadecimal support
         start_index = self.current
+        if self.peek() in ("+","-"):
+            self.advance()
 
+
```
Comment (tusharsadhwani) [inline src/json5kit/__init__.py]:
```suggestion
```

File: src/json5kit/__init__.py [hunk 2]
```diff
@@ -279,6 +279,8 @@ def parse_number(self) -> tuple[str, float]:
         # TODO: exponent syntax support
         # TODO: Hexadecimal support
         start_index = self.current
+        if self.peek() in ("+","-"):
```
Comment (tusharsadhwani) [inline src/json5kit/__init__.py]:
```suggestion
        if self.peek() in ("+", "-"):
```
